core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -367,21 +367,12 @@ trait WhiskActionsApi extends WhiskCollectionAPI with SequenceActions with Refer
                 l.memory getOrElse MemoryLimit(),
                 l.logs getOrElse LogLimit())
         } getOrElse ActionLimits()
-        // This is temporary while we are making sequencing directly supported in the controller.
```

Comment:
I suggest we don't yet do this - instead we continue to ignore sequence parameters until schema migration is resolved. Once we give up the invariant we wont be able to disambiguate real vs system injected parameters.